src/main/java/mezz/jei/ingredients/IngredientListElementInfo.java

```diff
@@ -143,4 +155,17 @@ public String getResourceId() {
 	public IIngredientListElement<V> getElement() {
 		return element;
 	}
+
+	//I needed the "V" to get all of the element -> itemStack flow to work.
```

Comment:
Instead of using the cheat itemstack, you can use the ingredient directly.
You can move this function out of this class and in to the `IngredientSorter.java` file by making the function static with a `<V>` like this:
```java
public static <V> ItemStack getItemStack(IIngredientListElementInfo<V> ingredientInfo) {
    IIngredientListElement<V> element = ingredientInfo.getElement();
    V ingredient = element.getIngredient();
    if (ingredient instanceof ItemStack) {
        return (ItemStack) ingredient;
    }
    return ItemStack.Empty;
}
```